### DISCUSSION TITLE
Pagina bloccata: L'accesso a "" non è consentito in base alla configurazione corrente.

### DISCUSSION THREAD

**Discussion by nicovoni** [SafeExamBrowser/seb-win-refactoring#1016] [Q&A] [answered]
After failing with the reset utility https://github.com/SafeExamBrowser/seb-win-refactoring/issues/956 I created a new configuration (I just set the start url) which I exported as a file and subsequently started on the PC stations. Something didn't work and I receive the following notice of which I attach a photo.

![20241105_110215](https://github.com/user-attachments/assets/7efb7704-6a24-4418-a47c-45ecac5ee085)

Then I recreated the same configuration from a new PC on which I had never started seb. This time everything worked correctly. So I have a doubt whether the reset utility actually performed a reset.

**Comment by dbuechel** [comment, marked as answer]
The URL filtering mechanism has nothing to do with the reset utility. If you receive the above message, then you must have enabled the URL filtering (see https://safeexambrowser.org/windows/win_usermanual_en.html#NetworkPane > Activate URL filtering).

Thus, you can simply disable it or otherwise you'll need to make sure you have set all the required rules correctly. Activating the live application log can help with this, see tab Security > Allow access to application log.
